test/unit/presubmit_test.go

```diff
@@ -43,23 +42,25 @@ func TestMainFunc(t *testing.T) {
 			contains("-- -short -race -count 1 ./..."),
 			contains("UNIT TESTS PASSED"),
 		},
-	}, {
-		name: `main --integration-tests`,
-		stdout: []check{
-			contains("RUNNING INTEGRATION TESTS"),
-			contains("Running integration test test/e2e-tests.sh"),
-			contains(fmt.Sprintf("go run knative.dev/test-infra/tools/kntest/cmd/kntest@latest"+
-				" kubetest2 gke --max-nodes=1 --machine=e2-standard-2 "+
-				"--enable-workload-identity --cluster-version=latest "+
-				"--extra-gcloud-flags --enable-stackdriver-kubernetes "+
-				"--no-enable-ip-alias --no-enable-autoupgrade "+
-				"--addons=NodeLocalDNS "+
-				"--test-command=%s/test/e2e-tests.sh "+
-				"--run-tests --extra-gcloud-flags= --logging=NONE "+
-				"--monitoring=NONE", rootDir)),
-			contains("INTEGRATION TESTS PASSED"),
-		},
-	}}
+	},
+	//{
+	// 	name: `main --integration-tests`,
+	// 	stdout: []check{
+	// 		contains("RUNNING INTEGRATION TESTS"),
+	// 		contains("Running integration test test/e2e-tests.sh"),
+	// 		contains(fmt.Sprintf("kubetest2 gke "+
+	// 			"--num-nodes=1 --machine=e2-standard-2 "+
+	// 			"--enable-workload-identity --cluster-version=latest "+
+	// 			"--extra-gcloud-flags --enable-stackdriver-kubernetes "+
+	// 			"--no-enable-ip-alias --no-enable-autoupgrade "+
+	// 			"--addons=NodeLocalDNS "+
+	// 			"--test-command=%s/test/e2e-tests.sh "+
+	// 			"--run-tests --extra-gcloud-flags= --logging=NONE "+
+	// 			"--monitoring=NONE", rootDir)),
+	// 		contains("INTEGRATION TESTS PASSED"),
+	// 	},
+	// }}
```

Comment:
Should this be deleted or changed?

Reply:
Hmm, so kntest prints the kubetest2 args to stdout and allowed this test to ensure the flags were tested here. The upstream kubetest2 doesn't do this so the test is broken. I might be able to fix that upstream and reenable this test.

Reply:
I'd say just delete this assertion then

Reply:
Cool

Reply:
done